map/discovery_manager.cpp

```diff
@@ -0,0 +1,64 @@
+#include "map/discovery_manager.hpp"
+
+namespace
+{
+std::string GetQuery(discovery::ItemType const type)
+{
+  switch (type)
+  {
+  case discovery::ItemType::Hotels: return "hotel ";
+  case discovery::ItemType::Attractions: return "attraction ";
+  case discovery::ItemType::Cafe: return "cafe ";
+  case discovery::ItemType::LocalExperts:
+  case discovery::ItemType::Viator: ASSERT(false, ()); return "";
+  }
+}
+}  // namespace
+
+namespace discovery
+{
+Manager::Manager(Index const & index, search::CityFinder * const cityFinder, APIs const & apis)
```

Comment:
Why not to pass cityFinder by const ref?

Reply:
ok